site/app/models/gradeable/Gradeable.php

```diff
@@ -2272,6 +2260,17 @@ public function isGradeInquiryEnded() {
         return false;
     }
 
+    /**
+     * Checks if the grade-inquiry settings are valid
+     * @return bool
+     */
+    public function isGradeInquirySettingsValid() {
+        if ($this->isTaGradeReleased() && $this->grade_inquiry_allowed && $this->grade_inquiry_start_date <= $this->grade_inquiry_due_date) {
+            return true;
+        }
+        return false;
```

Comment:
This can be simplified as follows:
```suggestion
        return $this->isTaGradeReleased() && $this->grade_inquiry_allowed && $this->grade_inquiry_start_date <= $this->grade_inquiry_due_date;
```

Reply:
This was a convention that was the same as isGradeInquiryEnded/isGradeInquiryYetToStart. I changed it in the next commit